arcaflow_plugin_sysbench/sysbench_schema.py

```diff
@@ -376,10 +376,15 @@ class LatencyAggregates:
         schema.name("Maximum"),
         schema.description("Maximum Latency"),
     ]
-    P95thpercentile: typing.Annotated[
+    percentile: typing.Annotated[
+        int,
+        schema.name("Percentile"),
+        schema.description("Reported percentile bucket"),
+    ]
+    percentile_value: typing.Annotated[
         float,
-        schema.name("95th Percentile"),
-        schema.description("95th percentile latency"),
+        schema.name("Percentile Value"),
+        schema.description("Percentile value"),
```

Comment:
Is this just latency, or can this be something else? If it is just latency, then I'd say it'd be a good idea to make that clear.

In addition, it may look better to have this as a sub-object to nest these two values. Is there a reason you chose against that?

Reply:
Agreed, I can improve the metadata there.

I chose this structure to match the existing sysbench output as closely as possible. The sysbench command will output something like:

```
95th percentile:            158.63
```

The new output structure still has both of these under a `Latency` parent.

```
output_data:
  sysbench_output_params:
    Primenumberslimit: 500000.0
    totaltime: 0.965
    totalnumberofevents: 9.0
    Numberofthreads: 2.0
    Validationchecks: 'off'
  sysbench_results:
    CPUspeed:
      eventspersecond: 9.31
    Latency:
      avg: 193.13
      min: 191.7
      max: 195.75
      percentile: 99
      percentile_value: 196.89
      sum: 1738.19
    Threadsfairness:
      events:
        avg: 4.5
        stddev: 0.5
      executiontime:
        avg: 0.8691
        stddev: 0.1
```